uni/lib/view/bug_report/widgets/form.dart

```diff
@@ -233,16 +233,19 @@ class BugReportFormState extends State<BugReportForm> {
     setState(() {
       _isButtonTapped = true;
     });
-
-    final String bugLabel = bugDescriptions[_selectedBug] == null
-        ? 'Unidentified bug'
-        : bugDescriptions[_selectedBug]!.item2;
-
+    final faculty = await AppSharedPreferences.getUserFaculties();
+    final bugReport = BugReport(
+        titleController.text,
+        descriptionController.text,
+        emailController.text,
+        bugDescriptions[_selectedBug],
+        faculty
+    ).toMap();
```

Comment:
Please add the type for faculty.
Also, 'faculty' should be 'faculties' because it's a list of faculties